stdlib/testing/diff.go

```diff
@@ -460,6 +461,27 @@ func (t *DiffTransformation) diff(key flux.GroupKey, want, got *tableBuffer) err
 	return nil
 }
 
+// Arbitrary floating point tolerance
```

Comment:
The arbitrary nature of this number rubs me the wrong way, but I realize float precision is a hard thing to handle. Does this have impact on our customers?

Reply:
It's definitely arbitrary. I tried to pick some sane conservative defaults. Our end-to-end tests I think are the biggest consumers of `diff`. This will affect tables with very minor floating point differences, but it seems like that's what customers would expect in most circumstances.